### PR TITLE
Practice 4-3

### PR DESCRIPTION
**Pull request by tzatushevskaya**
- added some startup idea

### CODE REVIEW

File: public/Sjofn/practice/sjofn4.html [hunk 1]
```diff
@@ -25,5 +25,16 @@
         console.log(Math.sqrt(p));
     }
 
+    let goods = prompt("Enter the name of the object you want to own here:");
+
+    function randomizePrice(min, max) {
+        // min = Math.ceil(min);
+        // max = Math.floor(max);
```
Comment (eclimov) [inline public/Sjofn/practice/sjofn4.html]: Please, remove commented instructions.

File: public/Sjofn/practice/sjofn4.html [hunk 2]
```diff
@@ -25,5 +25,16 @@
         console.log(Math.sqrt(p));
     }
 
+    let goods = prompt("Enter the name of the object you want to own here:");
```
Comment (eclimov) [inline public/Sjofn/practice/sjofn4.html]: Please, move this line below your function.
**Reason:** functions should be defined near each other and shouldn't have any other instructions between them.

**Review by SandaPanteleimonov — APPROVED** [review]
Not bad!
I always admire you.

**Review by eclimov — APPROVED** [review]
Well done

**Pull request merged**